File: Unidad_1/01_TDA_Lenguaje_C/persona.c
#include "persona.h"
void mostrarPersona(persona *personas){
    printf("Mi nombre es: %s %s \nEdad: %i años\n",personas->nombre,personas->apellido,personas->edad);
}
void comer(persona *personas){
    printf("Mis acciones son: \ncomer");
}
void caminar(persona *personas){
    printf("\ncaminar");
}
void hablar(persona *personas){
    printf("\nhablar");
}
void dormir(persona *personas){
    printf("\ndormir\n");
}

File: Unidad_1/02_TDA_Lenguaje_JAVA/persona.java
public class persona{
    //atributos
    private String nombre;
    private String apellido;
    private int edad;
    private char genero;

    //metodo_constructor
    public persona(String parametro_nombre, String parametro_apellido, int parametro_edad, char parametro_genero){
        this.nombre = parametro_nombre;
        this.apellido = parametro_apellido;
        this.edad = parametro_edad;
        this.genero = parametro_genero;
    }
    public void comer(){
        System.out.println("Mis acciones son: \ncomer");
    }
    public void caminar(){
        System.out.println("caminar");
    }
    public void hablar(){
        System.out.println("hablar");
    }
    public void dormir(){
        System.out.println("dormir");
    }
    
    public void mostrarPersona(){
        System.out.println("Mi nombre es: "+this.nombre+" "+this.apellido+"\nEdad: "+this.edad+" años\nGenero M/F: "+this.genero);
    }
}